src/Crate/PDO/PDO.php

```diff
@@ -80,29 +89,27 @@ class PDO extends BasePDO implements PDOInterface
      * {@inheritDoc}
      *
      * @param string     $dsn      The HTTP endpoint to call
-     * @param null       $username Unused
+     * @param null       $username Username for basic auth
      * @param null       $passwd   Unused
      * @param null|array $options  Attributes to set on the PDO
      */
-    public function __construct($dsn, $username, $passwd, $options)
+    public function __construct($dsn, $username = null, $passwd = null, $options = [])
     {
         foreach (ArrayUtils::toArray($options) as $attribute => $value) {
```

Comment:
Don't think this cast is needed anymore

Reply:
It's a nice wrapper around someone potentially giving us a null value

Reply:
`(array)` works too then ;-) (produces an empty array when casting `null`)